src/OrbitGl/TrackManagerTest.cpp

```diff
@@ -115,14 +115,34 @@ TEST_F(TrackManagerTest, NonVisibleTracksAreNotInTheList) {
   EXPECT_EQ(kNumTracks - 1, track_manager_.GetVisibleTracks().size());
 }
 
-TEST_F(TrackManagerTest, FiltersAndVisibilityWorkTogether) {
+TEST_F(TrackManagerTest, FiltersAndTrackVisibilityWorkTogether) {
   CreateAndFillTracks();
   track_manager_.GetOrCreateThreadTrack(TrackTestData::kThreadId)->SetVisible(false);
   track_manager_.SetFilter("thread");
   track_manager_.UpdateTracksForRendering();
   EXPECT_EQ(kNumThreadTracks - 1, track_manager_.GetVisibleTracks().size());
 }
 
+TEST_F(TrackManagerTest, FiltersAndTrackTypeVisibilityWorkTogether) {
+  CreateAndFillTracks();
+  track_manager_.SetTrackTypeVisibility(Track::Type::kThreadTrack, false);
+  track_manager_.SetFilter("thread");
```

Comment:
Are the threads called "...thread..."? A constant for that would be preferred.

Reply:
Just in this test. Filtering relies on the name of the thread, and the thread here is named "example thread". It's defined in `TrackTestData.h`, but I explicitely didn't choose to use the constant defined there because this way I also verify that it works with partial matching of the thread name, not just the exact name.